src/core/AnimatedProceduralNode.js

```diff
@@ -0,0 +1,53 @@
+import AnimatedNode from './AnimatedNode';
+import invariant from 'fbjs/lib/invariant';
+import AnimatedValue from './AnimatedValue';
+
+export default class AnimatedProceduralNode extends AnimatedNode {
+  numberOfArgs;
```

Comment:
we already keep `inputNodes` so can't we just use `inputNodes.length` instead? Or maybe we don't need to keep reference to `inputNodes` and just have that lenght kept?